packages/trader/src/sass/app/_common/components/player.scss

```diff
@@ -0,0 +1,244 @@
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            height: 3.6rem;
+            padding-bottom: 0.4rem;
+
+            .controls__right {
+                width: 10.2rem;
+            }
+        }
+
+        &__time-wrapper {
+            margin: 0 1.6rem;
+        }
+
+        &__button {
+            background: none;
+            border: none;
+            padding: 0;
+            cursor: pointer;
+            width: 2.4rem;
+            height: 2.4rem;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+
+            svg {
+                margin: 0;
+            }
+        }
+    }
+
+    &__volume__wrapper {
+        position: relative;
+
+        &:before {
+            content: '';
+            display: block;
+            height: 1rem;
+            width: 4rem;
+            position: absolute;
+            right: -1rem;
+        }
+    }
+
+    &__volume-bar__wrapper {
+        width: 4rem;
+        height: 10.4rem;
+        position: absolute;
+        bottom: 2.5rem;
+        right: -1rem;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 0.8rem 0;
+        border-radius: $BORDER_RADIUS * 2;
+        background-color: var(--general-main-2);
+        box-shadow: 0 0.8rem 1.6rem 0 var(--shadow-menu);
+        transform: translateY(0);
+        transition: transform 0.25s ease, opacity 0.25s linear;
+        opacity: 0;
+
+        &--enter,
+        &--exit {
+            transform: translateY(1rem);
+            opacity: 0;
+        }
+        &--enter-done {
+            transform: translateY(0);
+            opacity: 1;
+        }
+    }
+
+    &__volume-bar {
+        position: relative;
+        width: 0.4rem;
+        height: 100%;
+        border-radius: $BORDER_RADIUS;
+        cursor: pointer;
+        background-color: var(--text-less-prominent);
+    }
+
+    &__volume-bar__filled {
+        position: absolute;
+        bottom: 0;
+        width: 0.4rem;
+        border-radius: $BORDER_RADIUS;
+        background-color: var(--brand-red-coral);
+        transition: all 0.3s linear;
+    }
+
+    &__progress-dot,
+    &__volume-dot {
+        width: 1.2rem;
+        height: 1.2rem;
+        border-radius: 50%;
+        background-color: var(--brand-red-coral);
+        cursor: pointer;
+    }
+
+    &__progress-dot {
+        position: absolute;
+        left: 97.7%;
+        bottom: -0.3rem;
+
+        &:hover {
+            bottom: -0.6rem;
+            width: 1.6rem;
+            height: 1.6rem;
+        }
+    }
+
+    &__volume-dot {
+        position: absolute;
+        left: -0.4rem;
+        bottom: 90%;
+    }
+
+    &__playback-rate__wrapper {
+        width: 6.2rem;
+        position: relative;
+
+        .playback-rate__icon {
+            position: absolute;
+            left: 0;
+        }
+
+        .dc-dropdown__select-arrow {
```

Comment:
let's nest all styles for .dc-dropdown under .dc-dropdown
.dc-dropdown {
    &__select-arrow {
    &__display {
        &--playback-rate {
        &-text {
and so on